docs/advanced-topics/checksum-support.rst

```diff
@@ -87,6 +87,10 @@ However, in some cases the image checksum may diverge, for example, if the image
 been updated. For best results, and to avoid your Docker image being deleted because of a registry's retention policy,
 use Docker images referred to by a tagged version or digest. The verification features available may vary between execution engines.
 
+Note: For Amazon ECR and GitHub Container Registry images, checksum gathering will only work with Docker images produced with
```

Comment:
This content looks like it would be good to wrap in the sphinx `note` format, e.g. 
```
.. note::

    For Amazon ECR ...
```
Which will make it look like
<img width="912" alt="Screen Shot 2022-03-21 at 9 46 37 AM" src="https://user-images.githubusercontent.com/36607471/159274178-d078e802-fae3-4390-bcd7-f33e911c6e0d.png">